### PR TITLE
update README

### PR DESCRIPTION
**Pull request by andys-github**
- added language code for the `environment variables` code-snippet
- fixed a grammatical issue

### CODE REVIEW

**Comment by andys-github** [comment]
Here's a screenshot for the change:

image

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (deniskaber) [inline README.md:1]: It seems that this extra empty line is not needed here. Can you pls remove it?

**Review by deniskaber — CHANGES_REQUESTED** [review]
Hey @andys-github! Thank you for the contribution! 
Could you pls address one small comment I left?

File: README.md (hunk 1 above)
Comment (andys-github) [reply README.md:1]: Sure thing, @deniskaber!

**Pull request merged**